lib/truffle/rbconfig.rb

```diff
@@ -200,11 +202,14 @@ module RbConfig
     'libtruffleruby'    => libtruffleruby,
     'libtrufflerubytrampoline' => libtrufflerubytrampoline,
     'MAKEDIRS'          => 'mkdir -p',
+    'MAJOR'             => major,
     'MKDIR_P'           => 'mkdir -p',
+    'MINOR'             => minor,
     'NULLCMD'           => ':',
     'OBJEXT'            => 'o',
     'optflags'          => optflags,
     'OUTFLAG'           => '-o ',
+    'PATCHLEVEL'        => RUBY_PATCHLEVEL.to_s,
```

Comment:
In case `RUBY_PATCHLEVEL` would become a String this wouldn't make a copy of it so `"#{RUBY_PATCHLEVEL}` seems safer.
RUBY_REVISION for instance changed from Integer to String.